js/main.js

```diff
@@ -0,0 +1,23 @@
+// !!!  ДЗ 16. Рекурсивне зведення в ступінь
+
+// ??? Реалізувати рекурсивну функцію, яка зводить число в ступінь.
+// ??? Ступінь передається як другий аргумент у функцію
+// ??? pow (num, degree)
+
+// !!! Рішення
+
+function pow(num, degree) {
+   if (degree === 0) {
+      return 0;
```

Comment:
```suggestion
      return 1;
```
Можете перевірити `console.log(5,0) // 1`